src/ui/components/shared/SharingModal/PrivacyDropdown.tsx

```diff
@@ -135,8 +111,8 @@ function useGetPrivacyOptions(
       </DropdownItem>,
       <div key="option-team">
         {workspaces
-          .filter(w => filterByWorkspaceType(recording, w))
-          .sort((a, b) => sortWorkspaces(recording, a, b))
+          .filter(w => w.name && !!w.isTest == !!recording.isTest)
+          .sort((a, b) => a.name!.localeCompare(b.name!))
```

Comment:
Can `name` ever be null/undefined? (If so this would throw)

Reply:
It can't be but I added `w.name` to the filter above to be safe.